### PR TITLE
Update copyright notice

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we update the copyright notice to the year 2022.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Review by armiol — APPROVED** [review]

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/Runtime.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+     *
+     * @param command the command to execute
+     * @return the command line output
+     * @throws IllegalStateException upon an execution error
+     */
+    fun execute(vararg command: String): String {
+        val outWriter = StringWriter()
+        val errWriter = StringWriter()
+
+        val process = ProcessBuilder(*command)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/Runtime.kt:63]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/dart/DartContext.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+open class DartContext(dartEnv: DartEnvironment, internal val project: Project)
+    : DartEnvironment by dartEnv
+{
+    /**
+     * Executes `pub` command in this [Exec] task.
+     *
+     * The Dart ecosystem uses packages to manage shared software such as libraries and tools.
+     * To get or publish Dart packages, the `pub` package manager is to be used.
+     */
+    fun Exec.pub(vararg args: Any) = commandLine(pubExecutable, *args)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/dart/DartContext.kt:44]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/fs/LazyTempPath.kt [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    override fun resolveSibling(other: String): Path = delegate.resolveSibling(other)
+
+    override fun relativize(other: Path): Path = delegate.relativize(other)
+
+    override fun toUri(): URI = delegate.toUri()
+
+    override fun toAbsolutePath(): Path = delegate.toAbsolutePath()
+
+    override fun toRealPath(vararg options: LinkOption?): Path = delegate.toRealPath(*options)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/fs/LazyTempPath.kt:111]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/Update.kt [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+        if (!sshConfigFile.exists()) {
+            val parentDir = sshConfigFile.canonicalFile.parentFile
+            parentDir.mkdirs()
+            sshConfigFile.createNewFile()
+        }
+        return sshConfigFile
+    }
+
+    /** Executes a command in the project [rootFolder]. */
+    private fun execute(vararg command: String): String = Cli(rootFolder).execute(*command)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/Update.kt:196]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+        updatePagesTask.configure {
+            dependsOn(copyJavadoc)
+        }
+    }
+
+    private fun TaskContainer.registerCopyJavadoc(allowInternalJavadoc: Boolean) {
+        val inputs = composeInputs(allowInternalJavadoc)
+        register(copyJavadoc, Copy::class.java) {
+            doLast {
+                from(*inputs.toTypedArray())
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt:151]: ignore

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/javascript/JsContext.kt [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    /**
+     * Executes `npm` command in a separate process.
+     *
+     * This [File] is used as a working directory.
+     */
+    fun File.npm(vararg args: String) = project.exec {
+
+        workingDir(this@npm)
+        commandLine(npmExecutable)
+        args(*args)
```
[the remark this answers is not in the document]
Comment (dmdashenkov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/javascript/JsContext.kt:54]: ignore

**Pull request merged**